### DISCUSSION TITLE
Relationship between node and unit not working

### DISCUSSION THREAD

**Discussion by fbozzolo** [spine-tools/SpineOpt.jl#485] [Ideas]
Hi all,
I'm trying to model an energy system in which I use a gas turbine that should take gas from a gas node and then produce electricity to inject it into the electricity node. It does produce electricity, but out of nowhere, because no flow is registered from the gas node. In fact, if I add a 'fuel price' to the gas, the system's total cost is still zero.

I hope somebody can orient me on this issue (It happened with other relationships, but I guess it is the same problem)

Best regards,
Fabio Bozzolo

**Comment by mihlema** [comment]
Dear Fabio, 
thanks for your question. Could you provide a minimum working example? (e.g. you could export your db to a .json file, and post the text here, or attach the .sqlite as a zip file).
My first intuition would be a possible lack of the `fix_ratio_out_in` (or other related) parameters, which e.g. determine the efficiency of a power plant. This parameter should be defined on a `unit__node__node` relationship, so in your case probably something like `gasunit__electricitynode__gasnode`.

**Reply by fbozzolo** [reply]
Thanks! actually that and all the other problems I had have been solved, I was adding the `fix_ratio_out_in` parameter wrong.

Best regards!
